### DISCUSSION TITLE
I can't change my app language.

### DISCUSSION THREAD

**Discussion by RichardKanshen** [rocksdanister/lively#1415] [Q&A]
My app language is set to English, but it for some reason only shows in japanese, and I can't change it (?)
![image](https://user-images.githubusercontent.com/73738591/194897600-f881323f-6776-4970-ae30-3aad54eaf668.png)

**Comment by rocksdanister** [comment]
<img width="614" alt="image" src="https://user-images.githubusercontent.com/17554161/194897958-c405ba86-6e9b-4a52-9df1-219e0a4cfe15.png">

Your system language is Japanese? if not then what is it?

Related https://github.com/rocksdanister/lively/issues/1383

**Reply by RichardKanshen** [reply]
My system language is currently Slovak, but I only have installed English besides that.

**Comment by rocksdanister** [comment]
Lets continue discussing here: https://github.com/rocksdanister/lively/issues/1416
